playbooks/roles/openvpn/templates/instructions.md.j2

```diff
@@ -151,6 +151,12 @@ It's preferable to configure Ubuntu using the OpenVPN plugin for NetworkManager.
 1. Tap the slider to connect to the OpenVPN server.
 1. Success! You can verify that your traffic is being routed properly by [looking up your IP address on Google](https://encrypted.google.com/search?hl=en&q=ip%20address). It should say *Your public IP address is {{ streisand_ipv4_address }}*.
 
+<a name="UDP-profiles"></a>
+### Alternate unified profiles for access via port UDP port 8757 ###
```

Comment:
This should use the `{{ openvpn_port_udp }}` variable so that the documentation will be correct when the default is changed.

"via port UDP port" -> "via UDP port"